frontend/src/components/Button/styles.css

```diff
@@ -4,18 +4,18 @@
   justify-content: center;
   padding: 14px 38px;
   font-size: var(--body-font-size);
-  border-radius: 10px;
+  border-radius: 5px;
   cursor: pointer;
-  transition: background 0.2s ease;
-  margin: 10px;
 
   &.primary {
     background: var(--brand);
     color: var(--primary-text-white);
     border: none;
 
     &:hover {
-      background: var(--brand-lighter);
+      background: var(--brand-gradient-bright);
+      transition: background 0.5s ease;
```

Comment:
wont this only apply in hovered state

Reply:
Yes, but the animation looks smoother, applying it in the normal state makes the button glitch